docker-compose.yml

```diff
@@ -1,195 +1,81 @@
-        JWT_PUBLIC_KEY_PATH: ${JWT_PUBLIC_KEY_PATH:-config/jwt/public.pem}
-        JWT_PASSPHRASE: ${JWT_PASSPHRASE:-1234}
-        PHP_INI_DIRECTIVES: ${PHP_INI_DIRECTIVES:-upload_max_filesize=250M; post_max_size = 250M; memory_limit=256M}
-        MESSENGER_TRANSPORT_DSN: ${MESSENGER_TRANSPORT_DSN:-amqp://ergonode:ergonode@rabbitmq:5672/%2f/messages}
-        MESSENGER_TRANSPORT_IMPORT_DSN: ${MESSENGER_TRANSPORT_IMPORT_DSN:-amqp://ergonode:ergonode@rabbitmq:5672/%2f/import}
-        MESSENGER_TRANSPORT_CORE_DSN: ${MESSENGER_TRANSPORT_CORE_DSN:-amqp://ergonode:ergonode@rabbitmq:5672/%2f/core}
-        MESSENGER_TRANSPORT_EXPORT_DSN: ${MESSENGER_TRANSPORT_EXPORT_DSN:-amqp://ergonode:ergonode@rabbitmq:5672/%2f/export}
-        MESSENGER_TRANSPORT_DOMAIN_DSN: ${MESSENGER_TRANSPORT_DOMAIN_DSN:-amqp://ergonode:ergonode@rabbitmq:5672/%2f/domain}
-        MESSENGER_TRANSPORT_CHANNEL_DSN: ${MESSENGER_TRANSPORT_CHANNEL_DSN:-amqp://ergonode:ergonode@rabbitmq:5672/%2f/chanel}
-        MESSENGER_TRANSPORT_SEGMENT_DSN: ${MESSENGER_TRANSPORT_SEGMENT_DSN:-amqp://ergonode:ergonode@rabbitmq:5672/%2f/segment}
-        MESSENGER_TRANSPORT_COMPLETENESS_DSN: ${MESSENGER_TRANSPORT_COMPLETENESS_DSN:-amqp://ergonode:ergonode@rabbitmq:5672/%2f/completeness}
-        MESSENGER_TRANSPORT_NOTIFICATION_DSN: ${MESSENGER_TRANSPORT_NOTIFICATION_DSN:-amqp://ergonode:ergonode@rabbitmq:5672/%2f/notification}
-x-php-messenger:
-    - &php-messenger
-        working_dir: /srv/app
-        image: ${CONTAINER_REGISTRY_BASE:-ergonode}/php:${IMAGE_TAG:-latest}
-        volumes:
-            - ./backend:/srv/app
-        environment:
-            <<: *php-environment
-        depends_on:
-            - consul
-            - postgres
-            - rabbitmq
-        command: bin/console messenger:consume --sleep=10 --memory-limit=128M --time-limit=3600 -vv import channel export core event segment
-        healthcheck:
-            disable: true
-        restart: on-failure
+
 services:
+
     postgres:
-        build:
-            context: "./"
-            target: postgres
-            <<: *ergonode-cache-from
-        image: ${CONTAINER_REGISTRY_BASE:-ergonode}/postgres:${IMAGE_TAG:-latest}
+        image: "postgres:10"
         volumes:
             - "./data/.postgres:/var/lib/postgresql/data:delegated"
+            - "./config/postgres/init.sql:/docker-entrypoint-initdb.d/init.sql"
         environment:
-            POSTGRES_USER: ${POSTGRES_USER:-postgres}
-            POSTGRES_PASSWORD: ${POSTGRES_PASSWORD:-123}
-            POSTGRES_DB: ${POSTGRES_DB:-postgres}
-            APP_USER: ${APP_USER:-ergonode}
-            APP_USER_PASSWORD: ${APP_USER_PASSWORD:-123}
-            APP_DB: ${APP_DB:-ergonode}
-            APP_TEST_DB: ${APP_TEST_DB:-ergonode_test}
+            POSTGRES_USER: "postgres"
+            POSTGRES_PASSWORD: "123"
+            POSTGRES_DB: "postgres"
         ports:
-            - "${EXPOSED_POSTGRES_PORT:-5432}:5432"
+            - "5432"
```

Comment:
Could we leave at least ports configurable? It's easier to set up multiple applications if needed.

Reply:
Ok